pyramid_okta/security/authentication.py

```diff
@@ -75,8 +76,16 @@ def unauthenticated_userid(self, request):
         :param request: <pyramid.request>
         :return: user_id
         """
-        credentials = utils.get_credentials(request)
-        if credentials is None:
-            return None
-        user_id = credentials['client_id']
-        return user_id
+        if 'id' in request.session:
+            resp = utils.validate_session(request.session['id'])
```

Comment:
I think we should treat unauthenticated as if the cookie contains a valid session token return the user_id for the session else None and should not check against bearer or basic auth credentials at all. I may be misunderstanding.